### ISSUE TITLE
Unfortunately, CalDAV Sync Adapter has stopped.

### ISSUE THREAD

**Issue by lleb** [gggard/AndroidCaldavSyncAdapater#33]
We run OwnCloud on a CentOS 6.x server.  the syncing works with Thunderbirds Lightning application, but refuses to function properly with Caldav Sync Free Beta.  I get either the above error or one of your earlier bugs, the wrong URL when I know it is 100% correct.

Thank you in advance.

**Comment by gggard** [comment]
Is it still the case in the last version ?
If yes, please give me your server type

**Comment by lleb** [comment]
On 3/14/13 5:48 PM, Gérald GARCIA wrote:

> Is it still the case in the last version ?
> If yes, please give me your server type
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/gggard/AndroidCaldavSyncAdapater/issues/33#issuecomment-14931506.
> 
> It is semi working.  I get sync one way even with that box unchecked.  
> We run OwnCloud server on a CentOS 6.3 box.  In order to edit anything 
> or change calenders I have to use a web browser so it is not very 
> efficient of a sync application if it only syncs one way.

The creator has stated that it should work, but we have yet to get it to 
function properly.  His last offer was to give me a beta, but I have yet 
to receive the beta to install and test.

**Comment by gggard** [comment]
Sync one way will stay until we are sure that the application is stable for all the servers. Enabling it with bug may lead to delete all the events for the server...
As it is working I close the bug

**Comment by tareko** [comment]
Sadly, this issue is still the case for me, using owncloud. I'm not sure how to try to debug the issue. I am using the f-droid installation.

**Comment by chspnk** [comment]
Same to me with owncloud 5

**Comment by tareko** [comment]
I finally tracked down the source of this problem.

I looked at my server logs (not an option for many/most) and spotted that one event was causing the crash. I edited the event and now it works fine.

Here is the broken event. I've changed the person's name for confidentiality:

```
BEGIN:VCALENDAR
VERSION:2.0
CALSCALE:GREGORIAN
PRODID:-//Ximian//NONSGML Evolution Calendar//EN
BEGIN:VEVENT
STATUS:CONFIRMED
DTSTAMP:20110111T193557Z
CREATED:20120525T214847Z
CLASS:PUBLIC
TRANSP:OPAQUE
SUMMARY:Tom Joad's birthday
DESCRIPTION:Tom Joad's birthday
DTSTART:20040111T050000Z
RRULE:FREQ=YEARLY
DTEND:20040111T053000Z
UID:20110111T193557Z-11826-1000-1-3144@courage
LAST-MODIFIED:20120525T214847Z
END:VEVENT
END:VCALENDAR
```

Here is the fixed event. The entire calendar now syncs successfully. The only change I made was to turn the event into an 'all day event':

```
BEGIN:VCALENDAR
VERSION:2.0
CALSCALE:GREGORIAN
PRODID:-//Ximian//NONSGML Evolution Calendar//EN
BEGIN:VEVENT
STATUS:CONFIRMED
DTSTAMP;VALUE=DATE-TIME:20130417T100112Z
CREATED:20120525T214847Z
CLASS:PUBLIC
TRANSP:OPAQUE
SUMMARY:Tom Joad's birthday
DESCRIPTION:Tom Joad's birthday
DTSTART;VALUE=DATE:20040111
RRULE:FREQ=YEARLY;BYYEARDAY=11;INTERVAL=1
DTEND;VALUE=DATE:20040112
UID:20110111T193557Z-11826-1000-1-3144@courage
LAST-MODIFIED;VALUE=DATE-TIME:20130417T100112Z
END:VEVENT
END:VCALENDAR
```

**Comment by timoberger** [comment]
This is fixed with commit f7ddbb81c5d1c5ed5fc51de267a2fa8f29d71454 and should be available with the next release (see #53)

**Comment by timoberger** [comment]
this looks pretty old.
feel free to reopen when needed.
